dist/common/scripts/scylla_ntp_setup

```diff
@@ -26,6 +26,7 @@ import argparse
 import subprocess
 import re
 import distro
+import shutil
```

Comment:
Needs to be added to fedora_python3_packages.

Reply:
It's standard library, already used on other scripts

Reply:
ok